addon/helpers/pipe.js

```diff
@@ -12,7 +12,7 @@ export function pipe(actions = []) {
         return curr(...args);
```

Comment:
I also just noticed this: If any function in the pipe returns `undefined`, the next function will receive the original arguments. I don't know if this is a problem, but it could lead to subtle bugs around the ordering of operations. Maybe use a unique tag instead of `undefined`?

```js
const first = new Object();

export function pipe(actions = []) {
  return function(...args) {
    return actions.reduce((acc, curr) => {
      if (acc === first) {
        return curr(...args);
      } 
      
      return curr(acc);
    }, first);
  };
}
```